classes/configuration_exception.php

```diff
@@ -14,6 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with the ocis repository for Moodle.  If not, see <http://www.gnu.org/licenses/>.
 //
+// ownCloud Infinite Scale for moodle Repository is build with the contribution of:
+// - Staatsinstitut für Schulqualität und Bildungsforschung
+// - JankariTech
+// - ownCloud - a kiteworks company.
```

Comment:
```suggestion
// - ownCloud - a Kiteworks company.
```

Is capital "K" used for Kiteworks?

Reply:
done